.teamcity/Gradle_Promotion/buildTypes/Gradle_Promotion_ReleaseSnapshotFromQuickFeedback.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <build-type xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" uuid="eeff4410-1e7d-4db6-b7b8-34c1f2754477" xsi:noNamespaceSchemaLocation="http://www.jetbrains.com/teamcity/schemas/2018.1/project-config.xsd">
-  <name>Release - Snapshot (from Quick Feedback)</name>
-  <description>Deploys the latest successful change on 'release' as a new snapshot</description>
+  <name>Release - Release Nightly (from Quick Feedback)</name>
```

Comment:
It is still a release snapshot. Why do we want to rename it to a release nightly? A nightly also includes publishing documentation, and the job will not.

Reply:
It could if we wanted it to. I think you are conflating some concepts.

A “nightly” is just a symbolic pointer to a snapshot, and now so is “release nightly”. There is nothing stopping us instating https://docs.gradle.org/release-nightly/userguide/userguide.html if we have the need.